### DISCUSSION TITLE
Bun

### DISCUSSION THREAD

**Discussion by knpwrs** [remix-run/remix#3671] [Proposals]
Has anyone taken a look at running Remix in [Bun](https://bun.sh/)? The benchmarks for React SSR are intriguing:

<img width="387" alt="image" src="https://user-images.githubusercontent.com/174864/177819997-487607b5-fb7b-4231-b578-cf108f8ac2b6.png">

Fireship video:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/FMhScnY0dME/0.jpg)](https://www.youtube.com/watch?v=FMhScnY0dME)

**Comment by sergiodxa** [comment]
There's a WIP repo with an adapter https://github.com/jacob-ebey/remix-bun
